packages/editor-ui/src/components/mixins/targetBlank.ts

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue';
+
+export const targetBlank = Vue.extend({
+	methods: {
+		addTargetBlank(html: string) {
```

Comment:
this does not need to be a mixin as it's not stateful, should just be added as a helper function